analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/EmptyCollectionsShouldNotBeEnumeratedTest.cs

```diff
@@ -25,33 +25,33 @@ namespace SonarAnalyzer.UnitTest.Rules.SymbolicExecution
     [TestClass]
     public class EmptyCollectionsShouldNotBeEnumeratedTest
     {
-        private static readonly DiagnosticDescriptor[] OnlyDiagnostics = new[] { EmptyCollectionsShouldNotBeEnumerated.S4158 };
+        private static readonly DiagnosticDescriptor[] OnlyDiagnostics = { EmptyCollectionsShouldNotBeEnumerated.S4158 };
+
+        private static readonly VerifierBuilder Builder = new VerifierBuilder<SymbolicExecutionRunner>()
+            .WithOnlyDiagnostics(OnlyDiagnostics)
```

Comment:
Shouldn't there be a ` WithOnlyDiagnostics(params DiagnosticDescriptor[] diagnostics)` overload?

Reply:
Yes, it is. I wanted to inline it while refactoring but forgot about it :)